tools/timelock_proposal/mcm_with_timelock.go

```diff
@@ -0,0 +1,149 @@
+package timelock_proposal
+
+import (
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/smartcontractkit/ccip-owner-contracts/tools/errors"
+	owner "github.com/smartcontractkit/ccip-owner-contracts/tools/gethwrappers"
+	"github.com/smartcontractkit/ccip-owner-contracts/tools/mcms_proposal"
+)
+
+var ZERO_HASH = common.BytesToHash([]byte{})
```

Comment:
https://github.com/ethereum/go-ethereum/blob/v1.14.8/core/types/block.go#L484

Zero hash can just be `common.Hash{}`